config/rhoai/binding_admin_roles.yaml

```
@@ -0,0 +1,15 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kueue-admin-rolebinding
```

```suggestion
  name: admin-rolebinding
```
should not the prefix "kueue-" be added here on the fly?

---

You're right, now that it's been rebased on top of #4, the prefix is added automatically.